app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionBlockModal/serviceVersionsBlockModal.scss

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$WIDTH-DEFAULT: 480px;
```

Comment:
Where are you using it?

Reply:
This variable is used in this file but a bit lower.